### ISSUE TITLE
TypeError: Cannot read property '$off' of undefined when rapidly mounting and unmounting FroalaEditor

### ISSUE THREAD

**Issue by Kylelem62** [froala/react-froala-wysiwyg#178]
**Expected behavior**
FroalaEditor gracefully unmounts when ReactDOM unmounts and destroys it, even if the editor has been created and immediately destroyed.
**Actual behavior**
```
froala_editor.min.js:7 Uncaught TypeError: Cannot read property '$off' of undefined
    at E.Bootstrap.destroy (froala_editor.min.js:7)
    at t.value (index.js:1)
    at t.value (index.js:1)
    at callComponentWillUnmountWithTimer (react-dom.development.js:17169)
    at HTMLUnknownElement.callCallback (react-dom.development.js:149)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:199)
    at invokeGuardedCallback (react-dom.development.js:256)
    at safelyCallComponentWillUnmount (react-dom.development.js:17176)
    at commitUnmount (react-dom.development.js:17553)
    at commitNestedUnmounts (react-dom.development.js:17584)
```
**Editor Version**
3.0.0
_and_
3.0.2
**Browser**
Chromium
Google Chrome
**Related Issues**
https://github.com/froala/react-froala-wysiwyg/issues/176
https://github.com/froala/wysiwyg-editor/issues/3504

**Comment by nikolap** [comment]
Experiencing this issue as well, with both 3.0.0 and 3.0.2. Reproducible on Firefox as well.

**Comment by casualuser** [comment]
fixed in 3.0.4

**Comment by sophiagiordano** [comment]
@casualuser the react-froala-wysiwyg still uses froala-editor version 3.0.3?

**Comment by sophiagiordano** [comment]
> fixed in 3.0.4

**Comment by sophiagiordano** [comment]
> fixed in 3.0.4

@casualuser the react-froala-wysiwyg package still uses froala-editor version 3.0.3?
